extensions.md

```diff
@@ -254,6 +254,20 @@ tracking allocations directly).
 **New server capability**: `memoryUsageProvider : bool`:
  - Signals that the server supports `$/memoryUsage` requests.
 
+## Reference Container
+{:.v16}
+
+When finding references to a symbol, it's often useful to know the name of the
+function or class in which the reference occurs. 
+
+**New property of `Location` object**: `containerName : string?`:
+ - Name of the function or class in which the reference occurs. Might be empty,
```

Comment:
could you clarify the difference between empty vs none/missing (if any)?

Reply:
Done: our current implementation actually never sends none/missing here if the `textDocument.references.container` capability is set, so I changed the type from `string?` to `string`.

Reply:
Sorry, you're absolutely right! I for some reason thought that the index would just return an invalid/empty symbol in this case, so we would fill the containerName with an empty string still, but that's not the case.

Hope the wording is clear now.